test/attr/attr_objc_typealias_inside_protocol.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -c -enable-objc-interop -parse-as-library %s -o /dev/null -disable-objc-attr-requires-foundation-module -module-name typealias_objc -emit-module -emit-module-path %t/typealias_objc.swiftmodule
+// RUN: %target-swift-ide-test -skip-deinit=false -print-ast-typechecked -source-filename %s -function-definitions=true -prefer-type-repr=false -print-implicit-attrs=true -explode-pattern-binding-decls=true -disable-objc-attr-requires-foundation-module -swift-version 4 -enable-source-import -I %S/Inputs -enable-swift3-objc-inference | %FileCheck %s
```

Comment:
I think you don't need any of `-swift-version 4 -enable-source-import -I %S/Inputs -enable-swift3-objc-inference`.

Reply:
@jrose-apple Ah ok, I have removed them now!